data/pb/BPA/CheckList.json

```diff
@@ -0,0 +1,24 @@
+{
+    "tenantId": "pb",
+    "moduleName": "BPA",
+    "CheckList": [
```

Comment:
Change the master name, It is not a checklist, this is for docType mapping master

Reply:
renaming checkList to DocTypeMapping is that ok ?